web/src/lib/components/album-page/albums-list.svelte

```diff
@@ -0,0 +1,287 @@
+<script lang="ts" context="module">
+  import { AlbumViewMode, albumViewSettings } from '$lib/stores/preferences.store';
+  import { goto } from '$app/navigation';
+  import type { OnShowContextMenuDetail } from '$lib/components/album-page/album-card';
+  import { AppRoute } from '$lib/constants';
+  import { createAlbum, deleteAlbum, type AlbumResponseDto } from '@immich/sdk';
+  import { get } from 'svelte/store';
+
+  export const handleCreateAlbum = async () => {
+    try {
+      const newAlbum = await createAlbum({ createAlbumDto: { albumName: '' } });
+      if (newAlbum) {
```

Comment:
What's the case that createAlbum won't return the new album but it doesn't throw an error?

Reply:
Indeed, it shouldn't